receiver/genericwebhookreceiver/config.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package genericwebhookreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/genericwebhookreceiver"
+
+import (
+	"go.opentelemetry.io/collector/config/confighttp"
+)
+
+// Config defines configuration for the Splunk HEC receiver.
```

Comment:
```suggestion
// Config defines configuration for the webhook receiver.
```

Reply:
nice catch, basing the structure on the hec receiver since they are functionally very similar. Going to clean this up and submit PR